docs/google/index.rst

```diff
@@ -10,3 +10,5 @@ of devices that have become public information.
     :maxdepth: 1
 
     devices
+    engine
```

Comment:
sorry; I know this is already merged but:

Now that there's headings in the TOC this seperate index page is a bit redundant and actually makes the TOC and TOC headings less useful. I propose removing this `docs/google.index.rst` and linking directly to `google/devices`, `google/engine`, and `google/specification` directly from `docs/index.rst`